Develop

mbaumgart opened the pull request:
I added translations and a test for the css 'transform' property, when I stumbled upon a bug that a value, which was already translated, would be translated again if the values would match again ('drehen' -> 'rotate' -> 'redate'), which should be fixed now.

The npm test reported it works, though I wasn't able to run the module via either grunt or gulp to test the integration, as I couldn't get around the error message 'unexpected reserved word' triggered at 'require' in the index.js of the module, but maybe I'm using it wrong (I'm new to postcss).

Commit:
add properties, values and test for transform

Commit:
fix multiple translation problem

to prevent false chain translations like 'drehen' -> 'rotate' -> 'redate', the loop to translate/replace values now breaks when the first replacement happened

timche commented:
Thanks for this PR. Looks really neat already :) I will have a look at it tomorrow.

mbaumgart commented:
When I thought about that code again, I noticed that this might cause an
issue when having multiple values like "background: green no-repeat;", as
it will translate only the first one with this PR. I'll think of a better
solution tomorrow. One probably has to mark somehow which parts of the
string were translated already, to prevent double translations.

2016-04-30 0:14 GMT+02:00 Tim Cheung notifications@github.com:

> Thanks for this PR, I will have a look at it tomorrow.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/timche/postcss-german-stylesheets/pull/13#issuecomment-215895775

timche commented:
A few more test cases would be ideal for this.

Commit:
fix problem with word boundaries and unicode ß

This solution now does accept brackets, multi values and prevents to translate only parts of words
